README.md

```diff
@@ -420,7 +420,7 @@ You can use Clarinet to deploy your contracts to the public Testnet environment
 evaluation on a blockchain. Use the following command:
 
 ```bash
-$ clarinet deploy --testnet
+$ clarinet contract publish --testnet
```

Comment:
In 0.28.x, this should be:

> $ clarinet contracts publish ...

Reply:
Good catch, thanks.
Both `clarinet contract publish --network` and `clarinet contracts publish --network` should work for deploying contracts. Depending on the action (new, publish, etc) being sent to the subcommand contract/contracts, using the singular or plural might be more or less intuitive, which is why I created an alias `contract` for the `contracts` subcommand.
We moved `clarinet publish` to `clarinet contracts publish` because we are about to introduce a new type of offchain components that will also be published / deploy.   
This change went live with 0.28.

Reply:
https://github.com/hirosystems/clarinet/blob/7e57cd2e0202b90a0d49b006c6407f815b4c24c9/src/frontend/cli.rs#L124